data/json/ui/spacebar/spacebar_body_and_health_block.json

```diff
@@ -0,0 +1,390 @@
+[
+  {
+    "id": "player_health_and_stasistik_block",
+    "type": "widget",
+    "style": "layout",
+    "label": "1_player_block",
+    "width": 58,
+    "height": 17,
```

Comment:
height does nothing in layout widgets, and can be removed.

Reply:
ok, all done

Reply:
all done